python-avd/pyavd/_utils/password_utils/password.py

```diff
@@ -1,17 +1,17 @@
 # Copyright (c) 2023-2024 Arista Networks, Inc.
 # Use of this source code is governed by the Apache License 2.0
 # that can be found in the LICENSE file.
-"""
-Used by Encrypt / Decrypt filters
-"""
+"""Used by Encrypt / Decrypt filters."""
+
 from __future__ import annotations
 
 import random
+from typing import Any
 
 from .password_utils import cbc_decrypt, cbc_encrypt
 
 
-def _validate_password_and_key(password: str, key: str) -> None:
+def _validate_password_and_key(password: Any, key: str) -> None:
```

Comment:
is it `Any` or maybe `byte | str`?

Reply:
Since we are testing if it is a string, it seems we don't really know. This is called from filters in jinja, so type checking is non-existing.